modules/reactor/doc/content/source/meshgenerators/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator.md

```diff
@@ -0,0 +1,40 @@
+
+- [!param](/Mesh/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator/sides_to_adapt): a vector parameter that can include up to six elements which lists the reference square side numbers to adaptively mesh. These entries must be non-repeating integers from 0 to 3 (see [Figure 1](#schematic)). The square sides are indexed counterclockwise starting from the top side.
+- [!param](/Mesh/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator/inputs): a vector parameter that includes the same number of elements as [!param](/Mesh/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator/sides_to_adapt). These elements are input square meshes whose boundary nodes will be queried and used to adaptively mesh a side of the square mesh being created. The input square meshes are generated by [`PatternedCartesianMeshGenerator`](/PatternedCartesianMeshGenerator.md) or [`PolygonConcentricCircleMeshGenerator`](/PolygonConcentricCircleMeshGenerator.md). Multiple meshes may be input, each one corresponding to the side index value specified in [!param](/Mesh/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator/sides_to_adapt). The same input mesh may be repeated multiple times.
+
+!media reactor/meshgenerators/Square.png
+      style=display: block;margin-left:auto;margin-right:auto;width:40%;
+      id=schematic
+      caption=A schematic drawing showing the square side indices in `CartesianConcentricCircleAdaptiveBoundaryMeshGenerator` object.
+
+The [!param](/Mesh/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator/square_size) provided here must be consistent with the [!param](/Mesh/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator/square_size) values of all the [!param](/Mesh/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator/inputs) generators.
+
+One major application of this object is to generate meshes for control drums. In that case, [!param](/Mesh/CartesianConcentricCircleAdaptiveBoundaryMeshGenerator/is_control_drum) can be set as true so that corresponding `MeshMetaData` can be generated for future use with control drum related objects.
+
+## Example Syntax
+
+!listing modules/reactor/test/tests/meshgenerators/cartesian_concentric_circle_adaptive_boundary_generator/square_2d.i block=Mesh
+
+Here, a simple square mesh stored in an Exodus file is used as the input of `CartesianConcentricCircleAdaptiveBoundaryMeshGenerator`. Only Side 0 of the generated mesh is set to adapt the input mesh (i.e., Side 2 of the input mesh), as shown in [Figure 2](#cart_adapt). Note that only Side 0 of the generated mesh matches the input mesh (Side 2).
+
+!media reactor/meshgenerators/cart_adapt.png
+      style=display: block;margin-left:auto;margin-right:auto;width:40%;
+      id=cart_adapt
+      caption=A schematic drawing showing the input and generated meshes of the example case.
```

Comment:
```suggestion
      caption=A schematic drawing showing the generated meshes of the example case.
```

Reply:
fix others pls if copy-pasted

Reply:
The input mesh is included in the figure. So I think the original caption should be correct.